mautic/mautic
# Upgrade report bundle to work with Symfony 3 forms

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Automated tests included? | N
| Related user documentation PR URL |  N/A
| Related developer documentation PR URL | N/A
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/7989
| BC breaks? | N/A
| Deprecations? | N/A

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Upgrades the Report Bundle to work with Symfony 3 forms.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Ensure that creating reports works
3. Ensure exporting reports work
4. Ensure that the detail view of a report shows the correct data
